components/SummaryBox/SummaryBox.tsx

```diff
@@ -110,6 +143,14 @@ const Container = styled.div`
 	border: 1px solid ${(props) => props.theme.colors.buttonStroke};
 `;
 
+const ButtonContainer = styled.div`
```

Comment:
Not a big fan of the absolute positioning here. Please don't use this.

Reply:
What do you think is the best approach to do it? Currently, the container uses `position:relative` and the buttons `position:absolute`. The "Create" button uses `bottom: 15px` and the 'Add/Edit whitelist addresses' uses `bottom: 70px` to move them to the bottom, one below to the other. And works fine.

Reply:
@saeta-eth I'll do it all good